bench/pick-properties.ts

```diff
@@ -0,0 +1,144 @@
+// @ts-nocheck
+
+import { bench } from "@arktype/attest";
+import { PickProperties } from "../lib";
+
+export const runPickPropertiesBenchmarks = (majorMinorVersion: string) => {
+  bench("[PickProperties] picks nothing on empty object", () => {
+    return {} as PickProperties<{}, never>;
+  }).types([
+    {
+      "4.5": 9,
+      "4.6": 9,
+      "4.7": 9,
```

Comment:
Example of improvement (from https://github.com/ts-essentials/ts-essentials/pull/397/commits/8540ddb44a3371c91d910c6456235924a3da0f1c)

<img width="997" alt="Screenshot 2024-05-19 at 19 05 22" src="https://github.com/ts-essentials/ts-essentials/assets/2991847/34fa8624-4448-4240-80a2-10a053aa964b">